src/components/common/Drops/styles.js

```diff
@@ -71,7 +73,22 @@ export default theme => ({
     }
   },
   hideOnSmall: {
-    display: 'none'
+    display: 'none',
+    position: 'relative'
+  },
+  dropLogo: {
+    display: props => (props.bottom ? 'none' : 'block'),
+    position: 'absolute',
+    height: '60px!important',
```

Comment:
Avoid using !important. Use more specificity instead.

Reply:
I agree with avoiding the importants, however there are their exceptions like this one in which I´m placed to avoid nesting too much, but I will look for the other alternative